src/helper/httpRequest.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const baseUrl = 'http://localhost:3000/v1/';
```

Comment:
This should be an environment variable, but we can leave these here for now and change it later